docs/usage.rst

```diff
@@ -55,6 +55,18 @@ Note that you can still implicitly provide a schema in SQL statements, e.g.:
     used.
 
 
+Timeout
+=======
+
+Crate-PDO uses HTTP requests to query CrateDB. If a query lasts longer than the specified
```

Comment:
I would say "Setting the PDO attribute PDO::ATTR_TIMEOUT will raise a timeout exception and cancel the HTTP request after the specified timeout has elapsed." but otherwise I think your suggested wording is good!